requirements/constraints.txt

```diff
@@ -28,11 +28,9 @@ django-storages==1.14
 # for them.
 edx-enterprise==4.3.2
 
-# 1. django-oauth-toolkit version >=2.0.0 has breaking changes. More details
-#    mentioned on this issue https://github.com/openedx/edx-platform/issues/32884
-# 2. Versions from 1.5.0 to 2.0.0 have some migrations related changes.
-#    so we're upgrading minor versions one by one.
-django-oauth-toolkit==1.6.2
+# django-oauth-toolkit version >=2.0.0 has breaking changes. More details
+# mentioned on this issue https://github.com/openedx/edx-platform/issues/32884
+django-oauth-toolkit==1.7.1
```

Comment:
[Nit] It would be better to constrain to `<2.0.0` in case a security patch gets released, like 1.7.2. That hasn't happened yet, but something to consider for future constraints.